frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -79,7 +83,7 @@ def _modify_page(self, obj):
     def _create_page(self, obj):
         db_page = self.model()
         db_page.fingerprint = obj.meta['fingerprint']
-        db_page.url = obj.url
+        db_page.url = obj.url_as_unicode
```

Comment:
We haven't discussed this. I did this because by converting the url to unicode, I was able to avoid some encoding issues with some of the DB's. Non-ascii bytes were causing some issues for some of the them.